examples/data-sources/snowflake_secrets/data-source.tf

```diff
@@ -27,7 +27,7 @@ output "like_prefix_output" {
 # Filtering (in)
 data "snowflake_secrets" "in" {
   in {
-    schema = "schema_fully_qualified_name"
+    schema = "snowflake_schema.test.fully_qualified_name"
```

Comment:
It shouldn't be quoted (it's a reference to another object, [here's](https://developer.hashicorp.com/terraform/tutorials/configuration-language/dependencies#manage-implicit-dependencies) a link to docs on implicit dependencies). We could add the actual resource (`resource "snowflake_schema" "test" { ...<fields>... }`), but we do this rarely, so I think there's no need.

Reply:
Thanks for the docs, i have changed it to be used as reference to another obj, so removed the quotes